### PR TITLE
Base layout svelte

### PR DESCRIPTION
**Pull request by RitaOak**
Base layout modifications to fix styling issues after the migration from stencil.js to svelte

### CODE REVIEW

File: src/components/MainLayout.svelte [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
-      </div>
-      <div class="column options" />
+  </div>
+  <div class="column is-three-fifths main-wrapper">
+    <div class="column main-content">
+      <slot />
     </div>
   </div>
+  <div class="column right-sidebar">
+    <h1>Options</h1>
```
Comment (brecke) [inline src/components/MainLayout.svelte:17]: options?

**Review by brecke — CHANGES_REQUESTED** [review]
I do not know why there is a new lock file if there are no changes to `package.json`. It doesn't make sense 🤷 to push one without the other, so let's skip it.

File: src/components/MainLayout.svelte (hunk 1 above)
Comment (RitaOak) [reply src/components/MainLayout.svelte]: This was a placeholder for further content. As of now, I've removed it

Comment (brecke) [reply src/components/MainLayout.svelte]: 👍🏻 Sometimes we need placeholders, especially if we are to replace them soon enough. I don't think that would be the case with this one

**Review by brecke — APPROVED** [review]
LGTM

**Pull request merged**